src/utils/price.ts

```diff
@@ -23,6 +23,9 @@ const TOKEN_WHITELIST = [
 
 // Gets ETHs price in USD using the DAI / WETH Uniswap V3 pool.
 export function getEthPriceInUSD(): BigDecimal {
+  // TODO: REVERT FOR MAINNET. GOERLI POOLS ARE INACCURATE
+  // ! hardcode ETH price 1250 USD
+  return BigDecimal.fromString("1250");
```

Comment:
Maybe we could update to 1500 if going to be testing soon =)

Reply:
🧠 🤝

Reply:
1550 ;) https://github.com/valorem-labs-inc/valorem-subgraph/commit/7480f4849e93de9ff47666b010f1c62a5d88499d